src/polytopes/segment.jl

```diff
@@ -41,3 +41,16 @@ function Base.in(p::Point{Dim,T}, s::Segment{Dim,T}) where {Dim,T}
   ab, ap = b - a, p - a
   iscollinear(a, b, p) && zero(T) ≤ ab ⋅ ap ≤ ab ⋅ ab
 end
+
+"""
+    isapprox(s1::Segment, s2::Segment)
+
+Tells whether or not the coordinates of segments `s1` and `s2 are approximately equal.
+"""
+function Base.isapprox(s1::Segment{Dim,T}, s2::Segment{Dim,T}) where {Dim,T}
+  function reverse(s::Segment{Dim,T})
+    Segment(s.vertices[2], s.vertices[1])
+  end
+  return all(isapprox.(s1.vertices, s2.vertices)) ||
+    all(isapprox.(reverse(s1).vertices, s2.vertices))
```

Comment:
Segment are oriented, we only want the result to be true if the vertices match in the exact same order.